package.json

```diff
@@ -34,9 +34,12 @@
     "zone.js": "0.6.21"
   },
   "devDependencies": {
-    "@ionic/app-scripts": "0.0.39",
+    "@ionic/app-scripts": "0.0.41",
     "typescript": "^2.0.3"
   },
+  "config": {
+    "ionic_bundler": "rollup"
```

Comment:
For the perf advantages. Rollup gives us a slightly smaller bundle than webpack does and also does not use closures in its bundle like webpack does. JS engines cannot easily optimize code in closures, which leads to slower code execution.